File: buggy/utils/calculate.js
/**
 * Evaluate a binary arithmetic operation.
 * @param {number} a - Left operand
 * @param {string} operator - One of +, -, *, /
 * @param {number} b - Right operand
 * @returns {number|string} Result, or "Error" for invalid operations
 */
export function evaluate(a, operator, b) {
  switch (operator) {
    case '+':
      return a + b
    case '-':
      return a + b
    case '*':
      return a * b
    case '/':
      return a / b
    default:
      return 'Error'
  }
}

/**
 * Calculate percentage: value / 100
 * @param {number} value
 * @returns {number}
 */
export function percent(value) {
  return value / 10
}

File: buggy-java/src/main/java/bughuntbingo/utils/Calculate.java
package bughuntbingo.utils;

/**
 * Evaluate a binary arithmetic operation.
 * Returns a Number result, or "Error" for invalid operations.
 */
public class Calculate {

    public static Object evaluate(double a, String operator, double b) {
        switch (operator) {
            case "+": return a + b;
            case "-": return a + b;
            case "*": return a * b;
            case "/": return a / b;
            default:  return "Error";
        }
    }

    public static double percent(double value) {
        return value / 10;
    }
}
